src/platyPS/platyPS.psm1

```diff
@@ -570,6 +570,8 @@ function New-ExternalHelp
 
         [System.Text.Encoding]$Encoding = $script:UTF8_NO_BOM,
```

Comment:
`Encoding` and `Encode` names are obscure. Maybe `InputEncoding` and `OutputEncoding`?
Maybe just use `Encoding` for both?